tensorflow_cloud/preprocess.py

```diff
@@ -71,7 +71,10 @@ def get_wrapped_entry_point(entry_point,
         'import tensorflow as tf\n',
         strategy,
         'tf.distribute.experimental_set_strategy(strategy)\n',
-        # Add user's code
+        # Add user's code.
+        # We are using exec here to execute the user code object.
+        # This will support use case where the user's program has a
+        # main method.
```

Comment:
As we discussed we should seek to minimize the diff between what the user writes and what we execute. Why not copy the user's code into our own entry point file?

Reply:
I left the `exec` call here as copying the user code and executing exec would pretty much do the same thing. `exec` will only be just as unsafe as copying and executing.